packages/docs/components/button.md

```diff
@@ -111,18 +111,22 @@ They pair well with Primary and Secondary buttons.
 </Visual>
 
 
-### Overlay
+### Dismiss
 
 <Description>
 
-Use Overlay Buttons when Primary or Secondary variants aren't visually suitable. These Buttons may only be used on top of a base-variant background color, a photo, or an illustration.
+The Dismiss Button should only be used for dismissing UI. Typically, this Button is included as part of a larger Odyssey component, but may be useful when constructing unique UIs.
+
+The Dismiss Button has unique padding and will inherit the text color of it's parent UI.
```

Comment:
It may be worth talking a bit about accessibility implications here, and how to handle focus state should be managed if this is used in a component that appears when a user triggers it.